snapshots/devmapper/snapshotter.go

```diff
@@ -111,7 +111,7 @@ func (s *Snapshotter) Stat(ctx context.Context, key string) (snapshots.Info, err
 		err  error
 	)
 
-	err = s.withTransaction(ctx, false, func(ctx context.Context) error {
+	err = s.store.WithTransaction(ctx, false, func(ctx context.Context) error {
```

Comment:
Luckily this file shouldn't need to change at all 🤣

Reply:
🤣